Formula/eris.rb

```diff
@@ -0,0 +1,25 @@
+class Eris < Formula
+  desc "Blockchain application platform CLI"
+  homepage "https://erisindustries.com"
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
+  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+
+  bottle :unneeded
```

Comment:
You mean i will need something like this:
```
  bottle do
    cellar :any_skip_relocation
    sha256 "42a60235f9f31d12b59d33867b4f74ac4e4fa979fd92f7066bd5720da2eb607b" => :el_capitan
    sha256 "7acb5fcbd1230614a630cb4c0fefaef4a40b66dba633e067a641c17a4d444db2" => :yosemite
    sha256 "7cc1cf98d45f0cfd43e0d822d60430670e58dbd40f221033b2a446f0f03bb681" => :mavericks
  end
```

Do you recommend to drop the statement?